app/views/shared/_user_mgt_buttons.html.slim

```diff
@@ -0,0 +1,12 @@
+- if current_user.admin? || current_user.id == locals[:user].id
+  = button_to "Edit Info for #{user_path(locals[:user].id)} (#{locals[:user].tone_name})",
```

Comment:
I am explcitly setting the method and I have not been able to quickly find the CSS incantation to get the link to display as a properly sized button, so I am going to leave this as-is.